frontend/packages/ceph-storage-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as _ from 'lodash';
+import { humanizeBinaryBytesWithoutB } from '@console/internal/components/utils';
+import { PrometheusResponse, DataPoint } from '@console/internal/components/graphs';
+
+export const getMetricType = (resource, metricType) => _.get(resource, ['metric', metricType], '');
+
+export const formatToShortTime = (timestamp: Date): string => {
+  const dt = new Date(timestamp);
+  // returns in HH:MM format
+  return dt.toString().substring(16, 21);
+};
+
+export const getGraphVectorStats: GetStats = (response, metricType, unit) => {
```

Comment:
you should use `getInstantVectorStats` from `graph/utils`. You also dont need to specify `GetStats` as it is already in `graph/utils` too

Reply:
@rawagner can't use the getInstantVectorStats as it is for a single value doesn't work when multiple values are being returned.

Also the same for GetStats as in my case multiple PrometheusResponse are being returned and that is returning only one.